demo/src/task-scheduler.ts

```diff
@@ -0,0 +1,40 @@
+// Faster setTimeout(fn, 0) implementation using postMessage API
+// Based on https://dbaron.org/log/20100309-faster-timeouts
+export class MicroTaskScheduler {
+  readonly messageName = 'zero-timeout-message';
+
+  private executionQueue: Function[];
+  private stopped: boolean;
```

Comment:
same here: `private stopped = true;`, and then you don't even have to specify the type. Typescript will see the `false` and automatically infer the `boolean` type for this property.